classes/Domain/Services/Authentication.php

```diff
@@ -52,13 +52,6 @@ public function userId(): int;
      */
     public function check(): bool;
 
-    /**
-     * Determine whether the user is a non-authenticated guest.
-     *
-     * @return bool
-     */
-    public function guest(): bool;
```

Comment:
Other than that the method isn't currently used, we probably don't need two methods for a similar thing.